viewing_party/main.py

```diff
@@ -0,0 +1,175 @@
+
+def create_movie(title, genre, rating):
+    """
+    This function returns a dictionary of movies using the passed parameters as keys. It will return "None" if any parameter is empty.
```

Comment:
Love that you practiced making docstrings for the functions!
One minor opportunity for improvement: the norm is usually to continue on a new line once the string gets really long.